projects/plugins/jetpack-boost/composer.json

```diff
@@ -0,0 +1,100 @@
+{
+	"name": "automattic/jetpack-boost",
+	"description": "Boost your WordPress site's performance, from the creators of Jetpack",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.1.0",
+	"authors": [
+		{
+			"name": "Automattic, Inc.",
+			"email": "support@jetpack.com"
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"require": {
+		"ext-json": "*",
+		"automattic/jetpack-autoloader": "2.10.3",
+		"automattic/jetpack-config": "1.4.6",
+		"automattic/jetpack-connection": "1.29.0",
+		"automattic/jetpack-device-detection": "1.4.2",
+		"automattic/jetpack-lazy-images": "1.5.0",
+		"automattic/jetpack-terms-of-service": "1.9.8",
+		"automattic/jetpack-tracking": "1.13.7",
+		"tedivm/jshrink": "1.4.0"
+	},
+	"require-dev": {
+		"10up/wp_mock": "^0.3.0",
+		"dealerdirect/phpcodesniffer-composer-installer": "*",
+		"automattic/jetpack-codesniffer": "^2.2",
+		"guzzlehttp/guzzle": "7.3.x",
+		"mockery/mockery": "1.3.x-dev",
+		"phpunit/phpunit": "^6",
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.1",
+		"automattic/wordbless": "dev-master"
+	},
```

Comment:
These dependencies could probably use cleanup.

* `dealerdirect/phpcodesniffer-composer-installer` and `automattic/jetpack-codesniffer` shouldn't be needed here, linting is done from the monorepo root.
* I don't see anything in the plugin using `guzzlehttp/guzzle`, and the version listed [won't work with PHP 5.6](https://github.com/Automattic/jetpack/runs/3204631808?check_suite_focus=true#step:13:428).
* No need to directly depend on `phpunit/phpunit`. If nothing else, `yoast/phpunit-polyfills` already pulls in an appropriate version.
* The listed version of `10up/wp_mock` [is not installable with PHP 5.6](https://github.com/Automattic/jetpack/runs/3204631808?check_suite_focus=true#step:13:425).
* While I don't think it's a problem to use WorDBless, note that our testing infrastructure already sets up WordPress's test environment if you'd rather just test within that.

Reply:
Leaving this unresolved for the WorDBless comment.

Otherwise removed or downgraded in https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025 . Not sure what'll break from downgrading WP_Mock.